backend/src/config/database.ts

```
@@ -0,0 +1,80 @@
+import { PrismaClient } from '@prisma/client';
+import { config } from './index';
+
+// Prisma Client singleton
+let prisma: PrismaClient;
+
+declare global {
+  var __prisma: PrismaClient | undefined;
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)